qt/qt_common/map_widget.cpp

```diff
@@ -367,29 +357,14 @@ void MapWidget::initializeGL()
     auto fmt = context()->format();
     if (context()->format().version() < qMakePair(3, 0))
     {
-      LOG(LINFO, ("OpenGL ES version is below 3.0, taking the OpenGL ES 2.0 path"));
-      m_apiOpenGLES3 = false;
-
-      constexpr const char* requiredExtensions[3] =
-        { "GL_EXT_map_buffer_range", "GL_OES_mapbuffer", "GL_OES_vertex_array_object" };
-      for (auto & requiredExtension : requiredExtensions)
-      {
-        if (context()->hasExtension(QByteArray::fromStdString(requiredExtension)))
-          LOG(LDEBUG, ("Found OpenGL ES 2.0 extension: ", requiredExtension));
-        else
-          LOG(LCRITICAL, ("A required OpenGL ES 2.0 extension is missing:", requiredExtension));
-      }
-      fmt.setProfile(QSurfaceFormat::CompatibilityProfile);
-      fmt.setVersion(2, 0);
+      CHECK(false, ("OpenGL ES2 is not supported"));
     }
     else
     {
       LOG(LINFO, ("OpenGL version is at least 3.0, enabling GLSL '#version 300 es'"));
```

Comment:
You won't need an `else` branch anymore, as there will be no alternative, unless you plan to add vulkan here.